ui/packages/tidb-dashboard-for-dbaas/public/test-iframe/parent.html

```diff
@@ -47,7 +47,7 @@
         <a href="#/slow_query">SlowQuery</a>
         <a href="#/keyviz">KeyViz</a>
         <a href="#/topsql">TopSQL</a>
-        <a href="#/overview">Metrics</a>
+        <a href="#/metrics">Metrics</a>
```

Comment:
```suggestion
        <a href="#/monitoring">Monitoring</a>
```

The name `Metrics` had been changed to `Monitoring` in this [PR](https://github.com/pingcap/tidb-dashboard/pull/1370), would you please update this?

Reply:
I will update it after I publish the new tidb-dashboard-for-dbaas, in published tidb-dashboard-for-dbaas 0.0.15, it is still `/metrics`.

Reply:
Let me publish tidb-dashboard-for-dbaas 0.0.16

Reply:
Got it!

Reply:
![image](https://user-images.githubusercontent.com/1284531/180936158-b6591c17-7d80-4501-aab6-6009dd945a9d.png)

done!